lib/status/status.go

```diff
@@ -557,3 +572,91 @@ const (
 	// roleTag is the name of the tag containing node role
 	roleTag = "role"
 )
+
+// FromAlertManager collects alerts from the prometheus alertmanager deployed to the cluster
+func FromAlertManager(ctx context.Context, dnsAddr string) ([]*models.GettableAlert, error) {
+	client, err := httplib.GetPlanetClient(httplib.WithLocalResolver(dnsAddr))
```

Comment:
This is an undesired implicit dependency we should avoid by passing the planet client in configuration. The client will be created by the caller where dns configuration is available (local environment has it as does the cluster metadata record).
Also, the cluster record that `FromCluster` receives already contains the cluster's DNS configuration.